### PR TITLE
Tweaks to Github docs

### PR DESCRIPTION
**Pull request by nicholsj**
A few smallish changes to:

- Try to make requirements for setting up a new GOV.UK repo clearer (some of these steps are regularly missed) by splitting this out into a separate page
- Fix heading structure
- Improve general readability and fix typos

### CODE REVIEW

File: source/manual/github.html.md [hunk 1]
```diff
@@ -34,46 +34,45 @@ There are several GOV.UK GitHub teams within alphagov, including:
...
-
-Users are removed from the GitHub organisation when their entry in [govuk-user-reviewer][govuk-user-reviewer] is deleted.
+In these last two cases, you must state:
 
-# GOV.UK repos
+- your role
+- which team you're in
+- why you need access
+- your GitHub username
+- which GitHub team(s) you should join ([see list](#govuk-teams))
```
Comment (AgaDufrat) [inline source/manual/github.html.md:51]:
```suggestion
- which GitHub team(s) you should join ([see list](#gov-uk-teams-in-github))
```
The `id` will change after the header is updated from `GOV.UK teams` to `GOV.UK teams in Github`

**Review by AgaDufrat — APPROVED** [review]
🤞 it helps

File: source/manual/github.html.md [hunk 2]
```diff
@@ -34,46 +34,45 @@ There are several GOV.UK GitHub teams within alphagov, including:
...
+## Removing access to GitHub
 
-When creating a new GOV.UK repo, you must ensure it:
+Users are removed from the GitHub organisation when their entry in [govuk-user-reviewer][govuk-user-reviewer] is deleted.
 
-- has a well-written README (see [READMEs for GOV.UK applications](/manual/readmes.html), or the [GDS Way guidance](https://gds-way.digital.cabinet-office.gov.uk/manuals/readme-guidance.html#writing-readmes) for general repositories)
-- is tagged with the [`govuk`](https://github.com/search?q=topic:govuk) topic
-- has [Dependency Review](/manual/dependency-review.html) and [CodeQL](/manual/codeql.html) scans in its CI pipeline
-- is added to the [repos.yml](https://github.com/alphagov/govuk-developer-docs/blob/main/data/repos.yml) file in the Developer Docs.
+## Creating and configuring a new GOV.UK repo
```
Comment (AgaDufrat) [inline source/manual/github.html.md:57]: I would almost make it a new separate page, it doesn't seem to come up in search otherwise.

Comment (nicholsj) [reply source/manual/github.html.md:57]: Thanks, yes I wondered that. Will have a go

**Pull request merged**